### PR TITLE
Use terser es6+ compatible for JS minification

### PR DESCRIPTION
**Pull request by sirreal**
Use https://github.com/webpack-contrib/terser-webpack-plugin for JS minification

This appears to be the direction webpack-contrib is taking with regards to the unmaintained es6+ uglify-es minimizer.

See:

* https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/262
* https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/298#issuecomment-409702935

This PR is mostly to raise awareness for now. We might like to wait to see how community-wide adoption goes before making the change:

https://www.npmjs.com/package/terser-webpack-plugin

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -111,23 +111,13 @@ function getWebpackConfig( { extensionName = '' } = {}, argv ) {
...
-							 * https://github.com/mishoo/UglifyJS2/issues/3010
-							 */
-							collapse_vars: false,
-						},
-						mangle: {
-							safari10: true,
-						},
+					terserOptions: {
 						ecma: 5,
+						safari10: true,
```
Comment (sirreal) [inline webpack.config.js]: According to [docs](https://github.com/fabiosantoscode/terser#minify-options), this is `mangle: { safari10: true }, output: { safari10: true }`. Do we prefer just the mangle safari option?

File: webpack.config.js [hunk 2]
```diff
@@ -111,23 +111,13 @@ function getWebpackConfig( { extensionName = '' } = {}, argv ) {
...
 					parallel: true,
 					sourceMap: Boolean( process.env.SOURCEMAP ),
-					uglifyOptions: {
-						compress: {
-							/**
-							 * Produces inconsistent results
-							 * Enable when the following is resolved:
-							 * https://github.com/mishoo/UglifyJS2/issues/3010
-							 */
-							collapse_vars: false,
```
Comment (sirreal) [inline webpack.config.js]: Should be fixed in terser.

**Comment by sirreal** [comment]
You can verify that the bug which caused us to disable `compress` is not present by observing console output in the app with `localStorage.debug = 'calypso:popover'`:

ok

See #23388 for the PR which originally disabled `compress`.

File: webpack.config.js (hunk 1 above)
Comment (blowery) [reply webpack.config.js]: we should be setting both. I don't think the output option was available when we added the mangle setting, but maybe we just missed it.

**Comment by sirreal** [comment]
[ICFY report](http://iscalypsofastyet.com/branch/?branch=update/use-es6compat-terser-minimizer) is in 😁 

```
Commit: 771df265c9b5463e33eadc364f8e1f43acb45e9d
Author: sirreal
At: 2018-08-07T16:12:07Z
Message: Update deps
Delta:
chunk                                                                                                          stat_size           parsed_size           gzip_size
accept-invite                                                                                                       +0 B                 -80 B  (-0.3%)      -22 B  (-0.3%)
accept-invite~checkout~jetpack-connect~signup                                                                       +0 B                 -62 B  (-0.4%)      -21 B  (-0.4%)
accept-invite~jetpack-connect~signup                                                                                +0 B                 -68 B  (-0.2%)      -20 B  (-0.2%)
account                                                                                                             +0 B                 -49 B  (-0.2%)      -11 B  (-0.1%)
account-close                                                                                                       +0 B                 -11 B  (-0.1%)       -2 B  (-0.0%)
account-recovery                                                                                                    +0 B                 -24 B  (-0.1%)      -16 B  (-0.2%)
account~account-close~async-load-design-blocks~async-load-post-editor-media-modal~happychat~help~me~~f1ee5101       +0 B                -172 B  (-0.5%)      -65 B  (-0.7%)
account~account-close~happychat~help~me~notification-settings~privacy~purchases~security                            +0 B                 -92 B  (-0.1%)      -26 B  (-0.2%)
account~ads~async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-~bdd03cbc       +0 B                  -5 B  (-0.0%)      +10 B  (+0.2%)
account~async-load-design~async-load-design-gutenberg-components~async-load-design-playground                       +0 B                  -8 B  (-0.1%)       -5 B  (-0.1%)
ads                                                                                                                 +0 B                 -26 B  (-0.1%)       -9 B  (-0.1%)
async-load-blocks-calendar-popover                                                                                  +0 B                 -14 B  (-0.2%)       +0 B
async-load-blocks-calendar-popover~comments~post-editor                                                             +0 B                 -43 B  (-0.2%)      -15 B  (-0.3%)
async-load-blocks-inline-help-popover                                                                               +0 B                 -53 B  (-0.2%)      -13 B  (-0.2%)
async-load-blocks-inline-help-popover~help                                                                          +0 B                -100 B  (-0.1%)      -23 B  (-0.1%)
async-load-blocks-reader-full-post                                                                                  +0 B                 -67 B  (-0.2%)      -25 B  (-0.2%)
async-load-components-community-translator                                                                          +0 B                  -2 B  (-0.0%)       +3 B  (+0.1%)
async-load-components-happychat                                                                                     +0 B                  -1 B  (-0.0%)       -1 B  (-0.1%)
async-load-components-happychat~happychat                                                                           +0 B                 -47 B  (-0.4%)      -15 B  (-0.4%)
async-load-components-web-preview-component                                                                         +0 B                 -46 B  (-0.1%)       -4 B  (-0.0%)
async-load-components-web-preview-component~reader                                                                  +0 B                -282 B  (-0.2%)      -91 B  (-0.2%)
async-load-components-webpack-build-monitor                                                                         +0 B                  -5 B  (-0.2%)       +1 B  (+0.1%)
async-load-design                                                                                                   +0 B                -197 B  (-0.1%)      -50 B  (-0.1%)
async-load-design-blocks                                                                                            +0 B                -957 B  (-0.1%)     -357 B  (-0.2%)
async-load-design-blocks~async-load-post-editor-media-modal~media~notification-settings~people~post-~4d24709c       +0 B                 -64 B  (-0.4%)      -34 B  (-0.9%)
async-load-design-blocks~checkout~domains~purchases~woocommerce                                                     +0 B                 -13 B  (-0.1%)       -1 B  (-0.0%)
async-load-design-gutenberg-components                                                                              +0 B                 -31 B  (-0.2%)      -13 B  (-0.3%)
async-load-design-playground                                                                                        +0 B                -183 B  (-0.1%)      -43 B  (-0.1%)
async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-load-design-~8686940d       +0 B                 -19 B  (-0.1%)       -4 B  (-0.1%)
async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-load-design-~94b1095e       +0 B                 -20 B  (-0.1%)       -8 B  (-0.2%)
async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-load-design-~df281390       +0 B                 -17 B  (-0.1%)       -7 B  (-0.2%)
async-load-design~async-load-design-blocks~async-load-design-playground~checkout~domains~purchases~w~ea89a787       +0 B                 -54 B  (-0.0%)      -28 B  (-0.1%)
async-load-design~async-load-design-gutenberg-components~async-load-design-playground                               +0 B                -105 B  (-0.1%)      -25 B  (-0.1%)
async-load-design~async-load-design-playground~google-my-business                                                   +0 B                 -21 B  (-0.1%)       -1 B  (-0.0%)
async-load-design~async-load-design-playground~plugins                                                              +0 B                 -66 B  (-0.1%)       -6 B  (-0.0%)
async-load-docs-selectors                                                                                           +0 B                  -6 B  (-0.1%)       +1 B  (+0.1%)
async-load-extensions-woocommerce-app-store-stats                                                                   +0 B                 -48 B  (-0.2%)      -12 B  (-0.2%)
async-load-extensions-woocommerce-app-store-stats-listview                                                          +0 B                 -11 B  (-0.1%)       -5 B  (-0.1%)
async-load-extensions-woocommerce-app-store-stats-referrers                                                         +0 B                 -69 B  (-0.3%)      -11 B  (-0.2%)
async-load-extensions-woocommerce-app-store-stats~async-load-extensions-woocommerce-app-store-stats-~a2cc2cf0       +0 B                 -44 B  (-0.2%)       -8 B  (-0.1%)
async-load-featured-image                                                                                           +0 B                  -9 B  (-0.2%)       -3 B  (-0.2%)
async-load-layout-guided-tours                                                                                      +0 B                -385 B  (-0.4%)     -151 B  (-0.7%)
async-load-layout-masterbar-drafts-popover                                                                          +0 B                 -13 B  (-0.1%)       +1 B  (+0.0%)
async-load-lib-abtest-test-helper                                                                                   +0 B                  -1 B  (-0.0%)       -1 B  (-0.1%)
async-load-my-sites-current-site-domain-warnings~domains~plans                                                      +0 B                  -7 B  (-0.0%)       -4 B  (-0.1%)
async-load-my-sites-guided-transfer                                                                                 +0 B                 -22 B  (-0.1%)       +4 B  (+0.1%)
async-load-my-sites-site-settings-section-export                                                                    +0 B                 -50 B  (-0.2%)       -2 B  (-0.0%)
async-load-my-sites-site-settings-section-import                                                                    +0 B                -123 B  (-0.2%)      -23 B  (-0.1%)
async-load-notifications                                                                                            +0 B                  -5 B  (-0.1%)       -3 B  (-0.2%)
async-load-post-editor-editor-author                                                                                +0 B                 -36 B  (-0.2%)       -7 B  (-0.1%)
async-load-post-editor-editor-discussion                                                                            +0 B                 -10 B  (-0.2%)       -3 B  (-0.2%)
async-load-post-editor-editor-location                                                                              +0 B                 -15 B  (-0.1%)       -1 B  (-0.0%)
async-load-post-editor-editor-post-formats-accordion                                                                +0 B                 -22 B  (-0.3%)       -2 B  (-0.1%)
async-load-post-editor-editor-seo-accordion                                                                         +0 B                 -10 B  (-0.2%)       -5 B  (-0.2%)
async-load-post-editor-editor-sharing-accordion                                                                     +0 B                  -6 B  (-0.0%)       +5 B  (+0.1%)
async-load-post-editor-media-modal                                                                                  +0 B                 -37 B  (-0.2%)       -6 B  (-0.1%)
async-load-post-editor-media-modal~media~post-editor                                                                +0 B                -189 B  (-0.2%)      -35 B  (-0.1%)
async-load-post-editor-media-modal~media~post-editor~sharing                                                        +0 B                  -1 B  (-0.0%)       +0 B
async-load-post-editor-media-modal~post-editor                                                                      +0 B                 -66 B  (-0.1%)      -23 B  (-0.2%)
async-load-reader-conversations-stream                                                                              +0 B                  -8 B  (-0.2%)       +5 B  (+0.3%)
async-load-reader-feed-stream                                                                                       +0 B                  -1 B  (-0.0%)       +2 B  (+0.1%)
async-load-reader-feed-stream~async-load-reader-site-stream                                                         +0 B                 -14 B  (-0.0%)       -2 B  (-0.0%)
async-load-reader-following-manage                                                                                  +0 B                 -86 B  (-0.2%)      -28 B  (-0.3%)
async-load-reader-following-manage~async-load-reader-search-stream                                                  +0 B                 -11 B  (-0.0%)       +9 B  (+0.1%)
async-load-reader-list-stream                                                                                       +0 B                 -43 B  (-0.4%)      -24 B  (-0.8%)
async-load-reader-search-stream                                                                                     +0 B                  -5 B  (-0.0%)       +4 B  (+0.1%)
async-load-reader-sidebar                                                                                           +0 B                 -83 B  (-0.3%)      -36 B  (-0.5%)
async-load-reader-site-stream                                                                                       +0 B                 -14 B  (-0.2%)       -8 B  (-0.3%)
async-load-reader-tag-stream-main                                                                                   +0 B                  -6 B  (-0.0%)       +3 B  (+0.1%)
build                                                                                                               +0 B               -4695 B  (-0.3%)    -1132 B  (-0.4%)
checklist                                                                                                           +0 B                 -22 B  (-0.1%)       -9 B  (-0.2%)
checklist~plans                                                                                                     +0 B                 -20 B  (-0.1%)       +2 B  (+0.0%)
checkout                                                                                                            +0 B                -224 B  (-0.1%)      -46 B  (-0.1%)
checkout~customize~plugins~purchases~settings~theme~themes                                                          +0 B                -107 B  (-0.4%)      -41 B  (-0.6%)
checkout~domains                                                                                                    +0 B                 -24 B  (-0.1%)       -7 B  (-0.1%)
checkout~domains~plans                                                                                              +0 B                 -98 B  (-0.3%)      -22 B  (-0.3%)
checkout~plans                                                                                                      +0 B                 -42 B  (-0.2%)       -9 B  (-0.1%)
checkout~purchases                                                                                                  +0 B                 -44 B  (-0.1%)      -18 B  (-0.3%)
comments                                                                                                            +0 B                -271 B  (-0.2%)      -80 B  (-0.3%)
comments~stats                                                                                                      +0 B                 -65 B  (-0.3%)      -30 B  (-0.5%)
concierge                                                                                                           +0 B                 -63 B  (-0.1%)      -17 B  (-0.1%)
customize                                                                                                           +0 B                 -19 B  (-0.1%)       -8 B  (-0.2%)
devdocs                                                                                                             +0 B                -115 B  (-0.3%)      -42 B  (-0.4%)
domain-connect-authorize                                                                                            +0 B                 -25 B  (-0.2%)       -7 B  (-0.2%)
domains                                                                                                             +0 B                -348 B  (-0.1%)      -98 B  (-0.2%)
domains~plans                                                                                                       +0 B                 -14 B  (-0.1%)       +1 B  (+0.0%)
domains~signup                                                                                                      +0 B                -211 B  (-0.2%)      -98 B  (-0.3%)
feature-upsell                                                                                                      +0 B                 -17 B  (-0.0%)       -9 B  (-0.1%)
google-my-business                                                                                                  +0 B                 -77 B  (-0.1%)      -28 B  (-0.2%)
google-my-business~stats                                                                                            +0 B                 -28 B  (-0.2%)       -8 B  (-0.2%)
gutenberg-editor                                                                                                    +0 B                  -4 B  (-0.3%)       -3 B  (-0.5%)
hello-dolly                                                                                                         +0 B                 -14 B  (-0.2%)       -8 B  (-0.3%)
help                                                                                                                +0 B                 -38 B  (-0.2%)      -14 B  (-0.2%)
jetpack-connect                                                                                                     +0 B                -219 B  (-0.2%)      -34 B  (-0.1%)
jetpack-connect~plans~signup                                                                                        +0 B                 -46 B  (-0.1%)       -9 B  (-0.1%)
jetpack-onboarding                                                                                                  +0 B                 -70 B  (-0.1%)      -29 B  (-0.2%)
login                                                                                                               +0 B                 -70 B  (-0.1%)      -33 B  (-0.2%)
mailing-lists                                                                                                       +0 B                  -1 B  (-0.0%)       +0 B
manifest                                                                                                            +0 B                  +0 B                +1 B  (+0.0%)
media                                                                                                               +0 B                 -87 B  (-0.3%)      -30 B  (-0.4%)
notification-settings                                                                                               +0 B                 -57 B  (-0.1%)      -22 B  (-0.2%)
people                                                                                                              +0 B                -273 B  (-0.2%)      -64 B  (-0.2%)
plans                                                                                                               +0 B                 -75 B  (-0.1%)       -9 B  (-0.1%)
plugins                                                                                                             +0 B                -269 B  (-0.1%)      -81 B  (-0.2%)
plugins~themes                                                                                                      +0 B                 -34 B  (-0.1%)       -9 B  (-0.1%)
post-editor                                                                                                         +0 B                -853 B  (-0.2%)     -369 B  (-0.3%)
post-editor~settings-writing~woocommerce                                                                            +0 B                 -56 B  (-0.3%)      -14 B  (-0.3%)
posts-custom                                                                                                        +0 B                  -4 B  (-0.1%)       -2 B  (-0.1%)
posts-custom~posts-pages                                                                                            +0 B                -170 B  (-0.1%)      -68 B  (-0.2%)
posts-custom~posts-pages~stats                                                                                      +0 B                 -21 B  (-0.1%)       -2 B  (-0.0%)
posts-pages                                                                                                         +0 B                 -37 B  (-0.1%)       -9 B  (-0.1%)
preview                                                                                                             +0 B                  -1 B  (-0.0%)       +2 B  (+0.1%)
privacy                                                                                                             +0 B                  -5 B  (-0.1%)       -3 B  (-0.1%)
purchases                                                                                                           +0 B                -168 B  (-0.1%)      -74 B  (-0.2%)
reader                                                                                                              +0 B                -294 B  (-0.3%)     -141 B  (-0.6%)
security                                                                                                            +0 B                 -81 B  (-0.1%)      -24 B  (-0.1%)
sensei                                                                                                              +0 B                  -4 B  (-0.1%)       -3 B  (-0.3%)
settings                                                                                                            +0 B                -222 B  (-0.1%)      -87 B  (-0.2%)
settings-discussion                                                                                                 +0 B                 -42 B  (-0.1%)      -26 B  (-0.3%)
settings-security                                                                                                   +0 B                 -63 B  (-0.1%)      -25 B  (-0.2%)
settings-traffic                                                                                                    +0 B                -113 B  (-0.1%)      -43 B  (-0.2%)
settings-writing                                                                                                    +0 B                -205 B  (-0.1%)      -80 B  (-0.2%)
settings~settings-discussion~settings-security~settings-traffic~settings-writing                                    +0 B                 -20 B  (-0.1%)       -9 B  (-0.1%)
sharing                                                                                                             +0 B                 -94 B  (-0.1%)      -19 B  (-0.1%)
signup                                                                                                              +0 B                -244 B  (-0.1%)      -74 B  (-0.1%)
signup~stats                                                                                                        +0 B                  -6 B  (-0.0%)       +0 B
stats                                                                                                               +0 B                -352 B  (-0.2%)     -106 B  (-0.2%)
theme                                                                                                               +0 B                 -35 B  (-0.2%)      -17 B  (-0.3%)
theme~themes                                                                                                        +0 B                 -80 B  (-0.1%)      -38 B  (-0.3%)
themes                                                                                                              +0 B                -188 B  (-0.2%)      -75 B  (-0.4%)
vendors~accept-invite~async-load-my-sites-site-settings-section-import~notification-settings~people~~c485e6cf       +0 B                -227 B  (-0.4%)      -19 B  (-0.1%)
vendors~account~checkout~comments~domains~signup~stats                                                              +0 B                 -86 B  (-0.5%)      -42 B  (-1.0%)
vendors~async-load-blocks-calendar-popover~async-load-design~async-load-design-blocks~async-load-des~a9850f66       +0 B                -114 B  (-0.1%)      +34 B  (+0.2%)
vendors~async-load-blocks-calendar-popover~async-load-design~async-load-design-blocks~async-load-des~aab69609       +0 B                -194 B  (-0.5%)     -112 B  (-1.5%)
vendors~async-load-components-web-preview-component                                                                 +0 B                 -16 B  (-0.1%)       -2 B  (-0.0%)
vendors~async-load-components-web-preview-component~async-load-design~async-load-design-blocks~async~cb7e2f60       +0 B                 -38 B  (-0.1%)      -40 B  (-0.3%)
vendors~async-load-design-blocks                                                                                    +0 B                 -96 B  (-0.5%)      -18 B  (-0.3%)
vendors~async-load-design-blocks~async-load-reader-following-manage~async-load-reader-search-stream~~fc38723e       +0 B                -271 B  (-0.5%)     -191 B  (-1.6%)
vendors~async-load-design-blocks~post-editor~wp-job-manager~zoninator                                            +2073 B  (+2.1%)       +678 B  (+1.6%)     +186 B  (+1.8%)
vendors~async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-load~3d27eab9    +2032 B  (+0.1%)      -2232 B  (-0.2%)     -863 B  (-0.3%)
vendors~async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-load~a41c1ab5       +0 B                 -26 B  (-0.1%)       +0 B
vendors~async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-load~a48a2cd6       +0 B                  -6 B  (-0.0%)       +0 B
vendors~async-load-design~async-load-design-blocks~async-load-design-gutenberg-components~async-load~ee4b4cae       +0 B                 -26 B  (-0.1%)       -7 B  (-0.1%)
vendors~async-load-design~async-load-design-playground                                                              +0 B                -237 B  (-0.4%)      -89 B  (-0.5%)
vendors~async-load-design~async-load-design-playground~google-my-business~woocommerce                               +0 B                 -60 B  (-0.1%)      -11 B  (-0.1%)
vendors~async-load-emoji-text                                                                                       +0 B                  -8 B  (-0.0%)       -5 B  (-0.0%)
vendors~async-load-lib-happychat-connection                                                                         +0 B                -181 B  (-0.3%)      -59 B  (-0.4%)
vendors~async-load-notifications                                                                                    +0 B                -452 B  (-0.4%)     -168 B  (-0.6%)
vendors~async-load-post-editor-media-modal~media~post-editor                                                        +0 B                 -16 B  (-0.1%)       -7 B  (-0.2%)
vendors~async-load-reader-following-manage~async-load-reader-search-stream                                          +0 B                -881 B  (-0.9%)     -396 B  (-2.0%)
vendors~build                                                                                                    +2073 B  (+0.1%)      -1963 B  (-0.2%)     -760 B  (-0.2%)
vendors~checkout                                                                                                    +0 B                 -36 B  (-0.3%)       -5 B  (-0.1%)
vendors~devdocs                                                                                                     +0 B                 -74 B  (-0.1%)       -2 B  (-0.0%)
vendors~post-editor                                                                                                 +0 B                -224 B  (-0.3%)      -70 B  (-0.3%)
vendors~post-editor~woocommerce                                                                                     +0 B               -1919 B  (-0.4%)     -428 B  (-0.3%)
vendors~purchases                                                                                                   +0 B                 -15 B  (-0.1%)       +2 B  (+0.0%)
vendors~security                                                                                                    +0 B                 -78 B  (-0.3%)      -35 B  (-0.4%)
vendors~settings-traffic                                                                                            +0 B                -657 B  (-0.4%)     -283 B  (-0.6%)
vendors~settings-writing                                                                                            +0 B                 -74 B  (-0.3%)      -60 B  (-0.9%)
vendors~woocommerce                                                                                                 +0 B                 -54 B  (-0.2%)      -34 B  (-0.4%)
vendors~wp-super-cache                                                                                              +0 B                 -12 B  (-0.1%)       +5 B  (+0.1%)
woocommerce                                                                                                         +0 B               -2770 B  (-0.3%)     -727 B  (-0.4%)
wp-job-manager                                                                                                      +0 B                -100 B  (-0.2%)       -1 B  (-0.0%)
wp-job-manager~wp-super-cache~zoninator                                                                             +0 B                 -33 B  (-0.2%)      -16 B  (-0.4%)
wp-super-cache                                                                                                      +0 B                -126 B  (-0.1%)      -38 B  (-0.2%)
zoninator                                                                                                           +0 B                -132 B  (-0.2%)      -63 B  (-0.4%)
```

Bytes are saved.

**Comment by sirreal** [comment]
Just to highlight:

```
build                                +0 B               -4695 B  (-0.3%)    -1132 B  (-0.4%)
```

File: webpack.config.js [hunk 3]
```diff
@@ -111,23 +111,13 @@ function getWebpackConfig( { extensionName = '' } = {}, argv ) {
...
-						},
-						mangle: {
-							safari10: true,
-						},
+					terserOptions: {
 						ecma: 5,
```
Comment (sirreal) [inline webpack.config.js]: `ecma` options are available for parse, compress and output. Would it make sense to keep all of these at the highest `ecma` version available, and let babel deal with output?

https://github.com/fabiosantoscode/terser#minify-options

**Review by blowery — APPROVED** [review]
This seems good! Just needs a rebase to pick up shrinkwrap changes.

**Comment by sirreal** [comment]
Rebased with fresh `npm run update-deps`.

**Comment by flootr** [comment]
Just a quick heads up: I just merged the `uglifyjs-webpack-plugin@1.3.0` update which seems to have a [terser option](https://github.com/webpack-contrib/uglifyjs-webpack-plugin#terser) now. Not sure if it's any better or worse or if it was even possible before. Only saw it while comparing the sources between 1.2.7 and 1.3.0.

**Comment by sirreal** [comment]
Thanks for the heads up @flootr. It's unclear to me what direction these packages will take or how the community will handle this. I'm not sure whether anyone has a clear idea how this will evolve.

Can you think of clear advantages one way or the other?

One strategy might be to swap as this PR does now, and reevaluate after a time to see how things have evolved.

**Comment by flootr** [comment]
> One strategy might be to swap as this PR does now, and reevaluate after a time to see how things have evolved.

I think I'd go with that approach as well.

**Comment by sirreal** [comment]
I've rebased, updated to latest `terser-webpack-plugin@1.0.1`. If there are no objections I'll merge when e2e tests report back positively.

**Comment by sirreal** [comment]
Jetpack e2e tests are reporting failures. @brbrr will have a look tomorrow to see if failures are related. If not, we'll merge then. Thanks everyone!

**Comment by sirreal** [comment]
Thanks for checking @brbrr!

**Pull request merged**